validator/testcases/javascript/instanceproperties.py

```diff
@@ -188,6 +207,14 @@ def set__exposedProps__(new_value, traverser):
             "scopes is dangerous, and has been deprecated. If objects "
             "must be exposed to unprivileged scopes, `cloneInto` or "
             "`exportFunction` should be used instead."),
+        signing_help="If you are using this API to expose APIs to content, "
+                     "please use `Components.utils.cloneInto`, or "
+                     "`Components.utils.exportFunction`. If you are using it "
+                     "for other purposes, please consider using a built-in "
+                     "message passing interface instead. Extensions which "
+                     "expose APIs to content will be required to go through "
+                     "at least one round of manual code review prior to "
+                     "signing.",
```

Comment:
Editors will be able to flag warnings they consider relatively safe to be ignored in future updates, regardless of the severity. If a reasonably trustworthy add-on is using these APIs, it will probably get a pass.